node/overseer/src/lib.rs

```diff
@@ -705,10 +721,6 @@ where
 			let _ = s.tx.send(FromOverseer::Signal(OverseerSignal::Conclude)).await;
 		}
 
-		if let Some(ref mut s) = self.collator_protocol_subsystem.instance {
```

Comment:
missing `stop` impl for `collator_protocol_subsystem` and `collation_generation_subsystem`

Reply:
Good catch! And yeah, this whole file could really do with being heavily macro'd; there's a ton of extremely repetitive, easy-to-miss code in it.

Reply:
e5baa5d